DigitalLearningSolutions.Data.Tests/Services/SelfAssessmentServiceTests.cs

```diff
@@ -1,633 +1,20 @@
 ﻿namespace DigitalLearningSolutions.Data.Tests.Services
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Transactions;
-    using Dapper;
-    using DigitalLearningSolutions.Data.Models.SelfAssessments;
+    using DigitalLearningSolutions.Data.DataServices.SelfAssessmentDataService;
     using DigitalLearningSolutions.Data.Services;
-    using DigitalLearningSolutions.Data.Tests.TestHelpers;
     using FakeItEasy;
     using NUnit.Framework;
-    using FluentAssertions;
-    using Microsoft.Data.SqlClient;
-    using Microsoft.Extensions.Logging;
 
     public class SelfAssessmentServiceTests
     {
-        private SelfAssessmentService selfAssessmentService;
-        private const int SelfAssessmentId = 1;
-        private const int CandidateId = 11;
-        private SqlConnection connection;
+        private ISelfAssessmentDataService selfAssessmentDataService = null!;
+        private ISelfAssessmentService selfAssessmentService = null!;

```

Comment:
Looks like this file isn't used anywhere, is it worth keeping it around or should it be deleted?

Reply:
I'm going to add stuff to it in 648, so I figured we might as well keep it around for now to prevent needing to create it.